### ISSUE TITLE
Redesign /about page

### ISSUE THREAD

**Issue by georgianachao** [mastodon/mastodon#21099]
### Pitch

Related issues/PRs

https://github.com/mastodon/mastodon/issues/20930 https://github.com/mastodon/mastodon/issues/20777 https://github.com/mastodon/mastodon/issues/20798 https://github.com/mastodon/mastodon/pull/19803

First of all, there are way too much redundant information on the page. Most of the information showed in the main column repeats what's already there in the left one. There's even more redundant designs is you disable trends for you instance, then you'll get two search elements, one on the left and one on the right, that have the exactly same function. The waste of space here is even more unacceptable when you CANT even log in on this page. It's so weird that you have so much space wasted (and left), but at the same time you have to click on 'sign in' and then jump to a separate `auth/sign_in` page to log in and you have to click on 'create account' to know whether this instance is open for registration or not. FYI, on mobile web UI clicking on 'create account' takes you to https://joinmastodon.org/servers rather than opens a page for account registration. Before v4.0.0 you can get ALL these information and sign in and create account on the same page. I thought Mastodon wants to keep users in-app all the time as now clicking on the time indicator button doesn't even bring you to the original page on the remote server but opens it on your own instance https://github.com/mastodon/mastodon/issues/19652 , yet users need a different page to login, i just cant understand the logic here.

<img width="1336" alt="Screen Shot 2022-11-20 at 12 32 50 am" src="https://user-images.githubusercontent.com/83417118/202853464-bb3155dd-dc0b-42db-a9e6-47b863bca90a.png">

Moreover, as many mentioned before, admins should be able to remove 'search' from the `/about` page as well. This feature can be easily exploited. Some instances have disabled all kinds of unauthenticated access/use a white list federation mode, but people can just open a different instance and search for users on instances they dont have access to. This can put users, especially those from a country/region where speech is under extreme censorship. The only way to disable search for logged out users is to set `DISALLOW_UNAUTHENTICATED_API_ACCESS` to true, but as i commented in https://github.com/mastodon/mastodon/pull/19803#issuecomment-1320886057, this variable does not do what it claims to do, it disallows public status and profiles as well and even generates an unauthenticated error for the /about page.

I just dont understand why everything needs to be mounted into the same layout. Yes you make things consistent but at the same time much worse. Even the logout button in the footer is gone and you have to do an extra click to log out. 

<img width="294" alt="Screen Shot 2022-11-20 at 1 17 51 am" src="https://user-images.githubusercontent.com/83417118/202855271-ca87d011-08fc-49ad-a600-98dd497ebf29.png">

### Motivation

This is necessary and beneficial for both new and existing users (especially considering there's a lot of space wasted and left). Existing users dont need to read the server rules every time they open the site, make sign in possible on the page, not on a separate one. New users should know whether the instance is open for registration or not and should be able to register on the page, not on the separate one. It can also lead to frustration when someone finds the timeline of one instance very interesting and later find it isnt open for registration. Make the closed registration information available on the page, not on a separate modal. The space should be utilized in a better way.

**Comment by ooozerooo** [comment]
yeah, it doesn't make any sense that people cant log in on the landing page and have to open a separate log in page to do so. If Mastodon wants to keep the 3 panel design (i dont like it at all tho), just make the sign in form available in the right panel.
